### DISCUSSION TITLE
Dynamic purchasing system and rectify announcement after award or multiple awards

### DISCUSSION THREAD

**Discussion by WillemJanVos** [OP-TED/eForms-SDK#659] [Q&A] [answered]
**Background**
Currently I am writing a user story in relation to a tender and a dynamic purchasing system (DAS). What I want to support in the tender platform is the possibility to award repeatedly and also to rectify the tender announcement if, for example, the term is extended or there is another change. In the current situation, contracting authorities place a tender containing the link to the DAS. Subsequently, contracting authorities create a new tender in order to be able to publish the award if a contract is awarded in the DAS. I now want to bring this together in 1 tender on the tender platform so that the contracting authority maintains an overview. This is currently lacking because the announcement of the tender and the announcement of the awards are published in different tenders.

**Questions:**
1) Does TED support that there can be a rectification of the announcement after an award or multiple awards within the same tender? 
2) Also, does anyone have experience facilitating a DAS in a tender platform? So bringing together publishing announcement and publishing awards in 1 tender if there is a DAS.

**Comment by YvesJo** [comment, marked as answer]
Hi,
Please get directly in touch with the helpdesk for that non-technical question: ted@publications.europa.eu
